File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.js
function solution(n) {
    // 자연수 N을 문자열로 변경 후 구분자로 자른 배열 반환
    var num = (n).toString().split('');
    var answer = 0;
    
    for (let i = 0; i < num.length; i++) {
        answer += parseInt(num[i]); // 문자 => 숫자
    }

    return answer;
}

File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        // 숫자 => 문자
        // String num = Integer.toString(n);
        String num = n + "";
        
        for (int i = 0; i < num.length(); i++) {
            char c = num.charAt(i);
            
            // char => String
            // answer += Integer.parseInt(String.valueOf(c));
            answer += Integer.parseInt(c + "");
        }
        
        return answer;
    }
}